crates/lgn-graphics-renderer/src/components/camera_component.rs

```diff
@@ -185,12 +188,15 @@ impl Default for CameraComponent {
             fov_y: Angle::from_radians(std::f32::consts::FRAC_PI_4),
             z_near: 0.01,
             z_far: 100.0,
+            render_object_id: None,
         }
     }
 }
 
-pub(crate) fn create_camera(mut commands: Commands<'_, '_>) {
-    commands.spawn().insert(CameraComponent::default());
+pub(crate) fn tmp_create_camera(mut commands: Commands<'_, '_>) {
+    commands
+        .spawn()
+        .insert_bundle((GlobalTransform::default(), CameraComponent::default()));
```

Comment:
The transform is inside the camera rig. I add a GlobalTransform because of the reflect system. It might not be needed but right now, the camera is not following the other entity setup (GlobalTransform).

Reply:
Why not query just the CameraComponent in reflect and grab the transform from the camera rig? Are we planning to switch to using GlobalTransform as camera transform?